apple/Sources/FerrostarMapLibreUI/Views/DynamicallyOrientingNavigationView.swift

```diff
@@ -24,78 +19,103 @@ public struct DynamicallyOrientingNavigationView<
     private var navigationState: NavigationState?
     private let userLayers: [StyleLayerDefinition]
 
-    var topCenter: TopCenter
-    var topTrailing: TopTrailing
-    var midLeading: MidLeading
-    var bottomTrailing: BottomTrailing
+    public var topCenter: (() -> AnyView)?
+    public var topTrailing: (() -> AnyView)?
+    public var midLeading: (() -> AnyView)?
+    public var bottomTrailing: (() -> AnyView)?
```

Comment:
Do we need to add these to the constructor? Or I guess there's an extension so maybe the idea is that you don't need to override(?) these very often so it's easier to just leave it to extension methods?

Reply:
Nope. To keep the constructors sane, I think these only get modified exclusively through the extension view modifier style.